Makefile

```diff
@@ -48,6 +48,10 @@ doc-serve: ## Serves documentation site
 
 # --- testing -------------------------------------------------------------------------------------
 
+.PHONY: test-build
+test-build: ## Build the test binary
+	$(DEBUG_ASSERTIONS) cargo nextest run --cargo-profile test-release --features concurrent,testing --filter-expr "not test(prove)" --no-run
```

Comment:
If we use `--no-run` here, why do we also need `filter-expr` to filter out some tests?

Reply:
We don't; I'm just bad at copy-pasting :) Thanks!